src/TableData.cpp

```diff
@@ -805,3 +809,169 @@ K parseArrowData(K char_array, K options)
 
   KDB_EXCEPTION_CATCH;
 }
+
+K readORCData(K orc_file, K options)

```

Comment:
Can we support null mapping and null bitmap?
I think so
Model off the new readParquetData / readArrowData / parseArrowData

Reply:
Created a task in Jira to support nulls - it seem requires changes in sources, tests and docs by itself
https://kxl.atlassian.net/browse/KXI-24287